parsers/OPENNEM.py

```diff
@@ -29,11 +29,11 @@
         'direction': 1,
     },
     'AUS-NSW->AUS-VIC': {
-        'region_id': 'VIC1->NSW1',
-        'direction': 1,
+        'region_id': 'NSW1->VIC1',
+        'direction': -1,
```

Comment:
```suggestion
        'direction': 1,
```
The reason is that the direction of the key (AUS-NSW->AUS-VIC) is the same as the region_id (NSW1->VIC1)